cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflowAction.java

```diff
@@ -49,8 +50,8 @@ public void destroy() {
     // No-op
   }
 
-  protected final WorkflowContext getContext() {
-    return context;
+  public WorkflowContext getContext() {
```

Comment:
but that will happen in the action that user writes, right? which would be extended from the `AbstractWorkflowAction`